.gitignore

```diff
@@ -32,3 +32,5 @@ project/.gnupg
 !project/.gnupg/secret.tar.enc
 out
 .vscode/
+
+yarn.lock
```

Comment:
We had a discussion about it on Discord today, fs2 has the same problem. @mpilquist and @ChristopherDavenport are for removing it. @mpilquist pointed out that it seems to be changing by itself which means what should be a release build could suddenly become a snapshot. It's probably tooling issues.

In our case, I don't think it's a _huge_ deal, this only showed up because I added a web socket client dependency to test out @RaasAhsan's PR on Node.js. We have no such dependencies in the Compile scope.

Reply:
I guess that's not too severe in this limited case, but it sounds highly problematic in a general case.  I don't see how builds can be repeatable without sharing the lock file.

Is it changing between compile and test?  Is it because of this?  https://github.com/scalacenter/scalajs-bundler/issues/259.

Reply:
That's a very good point about repeatable builds and thanks for finding that issue, maybe that's all we need to do to fix. I'm curious if this sways @ChristopherDavenport or @mpilquist.